Gemfile

```diff
@@ -53,7 +53,8 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
-  gem 'rspec-rails'
+  gem 'rails-controller-testing'
+  gem 'rspec-rails', '~> 6.0.0'
 end
 
```

Comment:
- [ ] `README.md` file: I think it would be better if you improve your `Getting Started` section by adding there all information needed by a usuer such as to `Setup`, your app, `access` the cloned directory, `install`, `use/run` your app, `run tests`,... This will make your documentation to be more descriptive and more professional. To get an example I suggest you refer this template.

- [ ] Also to correctly match the required README template, I suggest you order all your sections like in the template such as your `Future Features` section should come after the `Authors` section.